src/xml/report.html.xsl

```diff
@@ -1,62 +1,122 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
 
-<xsl:stylesheet 
-  xmlns:xsl="http://www.w3.org/1999/XSL/Transform" 

```

Comment:
I think that was my text editor doing that or something. I'll see if I can back that out